src/astro_image_display_api/interface_definition.py

```diff
@@ -57,41 +57,19 @@ class ImageViewerInterface(Protocol):
 
     # The methods, grouped loosely by purpose
 
-    # Methods for loading data
+    # Method for loading data
     @abstractmethod
-    def load_fits(self, file: str | os.PathLike) -> None:
+    def load(self, data: Any) -> None:
```

Comment:
What about the `**kwargs`? For example, FITS might need also the `ext` keyword to specific extension, but this keyword does not apply if one loads a pure numpy array, so we have to stay flexible here?

Reply:
I'd like to add `**kwargs` everywhere at the same time...